### ISSUE TITLE
Bug: Likes Count Not Updating Correctly upon refreshing the page in recipes_hub.js

### ISSUE THREAD

**Issue by DivyanshGarg380** [supriya46788/BakeGenuis-AI#138]
Bug 1: 

If a USER likes a recipe , the likes count increases by one as expected . When the user clicks refresh button , the likes count decreases by 1 . The hear symbol is still red so by pressing it again it decreases in count by 1 and turns grey . so total loss = 2.

My fix : 

Store the likes count in local storage and update accordingly. 

Bug 2 : 

The User profile pic , name and logout button miss styling . Its due to missing auth css file in the recipe_hub.html file.

My fix : 

Add the missing CSS file ( auth.css ) in the file after the auth.js is loaded in the file :)
